Assets/Scripts/CardController.cs

```diff
@@ -2,25 +2,41 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-public class CardController : MonoBehaviour
-{
-    public HandController hand;
+public class CardController : MonoBehaviour {
+    public HandController handController;
     public Color32 unselectedColor;
     public Color32 selectedColor;
-
+    public GameObject cardIndexPrefab;
+    CardIndexController cardIndexController;
     SpriteRenderer spriteRenderer;
-    bool selected;
+    bool isSelected;
 
     // Start is called before the first frame update
-    void Start()
-    {
-        spriteRenderer = GetComponent<SpriteRenderer>();
+    void Start () {
```

Comment:
Card will always generate a child gameobject CardIndex on Start().  On default, it is empty and does not display